ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -41,7 +43,14 @@ component:
   valve: "Component which meters the flow of water within a system or device."
   vane: "Component for guiding media flow."
   wheel: "Component used for transfer of heat from incoming to outgoing air streams. "
-descriptor:
+descriptor: 
```

Comment:
can the subfields defined below please be added in alphabetical order to allow for readability upon commit?

Reply:
ok, I have rearrange the order.